statsmodels/tools/testing.py

```diff
@@ -16,8 +16,10 @@ def strip_rc(version):
 
 
 def is_pandas_min_version(min_version):
-    '''check whether pandas is at least min_version '''
-    return StrictVersion((pandas.__version__[:6])) >= min_version
+    '''check whether pandas is at least min_version
+    '''
+    from pandas import __version__ as pversion
+    return StrictVersion(strip_rc(pversion)) >= min_version
```

Comment:
True. I'd think that it's a hard requirement to handle development versions of dependencies correctly in a project though. If you get test failures because your version checks say that ``0.17.0+githash == 0.17.0`` that would be quite annoying.